pkg/ruler/compat.go

```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
 	"time"
 
+	"github.com/cortexproject/cortex/pkg/querier/stats"
+
```

Comment:
nit: they are under the same package, we can probably remove this newline

Reply:
We should group imports from `cortexproject` together. Let's move them to the group in L29

Reply:
Done